drivers/adc/adc_smartbond_gpadc.c

```diff
@@ -101,6 +110,50 @@ static int adc_smartbond_channel_setup(const struct device *dev,
 	return 0;
 }
```

Comment:
What about `adc_smartbond_channel_setup()`?
Writing to registers will fail if we don't make it "PM aware", won't it?

Reply:
Unless I miss something, the mentioned API should not access any register. Control values are stored in variables that are actually applied when sampling is initiated.

Reply:
I thought that the variables were pointers to registers. Ignore my comment.